harness/tests/cli/test_user.py

```diff
@@ -47,3 +48,29 @@ def test_user_edit_no_fields(mock_die: mock.MagicMock) -> None:
         mock_die.assert_has_calls(
             [mock.call("No field provided. Use 'det user edit -h' for usage.", exit_code=1)]
         )
+
+
+@mock.patch("getpass.getpass")
```

Comment:
It shouldn't be too dangerous, but this code _can_ touch the actual filesystem via the TokenStore. For this reason, I recommend patching `determined.common.api.authentication.TokenStore`, too. I'm fine letting using the default mock class as its target (as is done when you don't pass a target to its class)

```python
@mock.patch("determined.common.api.authentication.TokenStore")
```
But since we have a fake `TokenStore`, you can also do

```python
@mock.patch("determined.common.api.authentication.TokenStore", util.MockTokenStore)
```

if you like. I'm not sure which I'd choose, myself. More complex or more brittle, seems like a toss-up. Including the option to help you get up to speed with Python `unittest.mock`.

Reply:
That's a great point. It better to avoid touching the filesystem in this test.